### PR TITLE
AMI: verify that enhanced network is enabled

### PR DESCRIPTION
**Pull request by amoskong**
Enhanced network was enabled by https://github.com/scylladb/scylla/commit/33309f66
This patch verifies the enhanced network is enabled in AMI instances by
checking the driver name.

Reference:
https://github.com/scylladb/scylla/commit/b8f40a2d
http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking

### CODE REVIEW

**Comment by slivne** [comment]
LGTM

File: scylla-artifacts.py [hunk 1]
```diff
@@ -508,9 +508,44 @@ def setup_release(self):
 
 
 class ScyllaInstallAMI(ScyllaInstallGeneric):
+    def check_used_driver(self, expected_driver=''):
+        """
+        Make sure VPS is enabled and enhanced network driver is used.
+        """
+        with open('/sys/class/net/eth0/address', 'r') as f:
+            address = f.read().strip()
+        result = process.run('curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/%s/' % address)
```
Comment (roydahan) [inline scylla-artifacts.py]: What is this address?

Comment (amoskong) [inline scylla-artifacts.py]: It's address of EC2 service, we also used this in scylla code.

**Comment by amoskong** [comment]
[Test artifact 1.7 job ](http://jenkins.cloudius-systems.com/job/scylla-1.7-ami-test-amostmp/8/label=muninn/artifact/latest/job.log)is good, I will merge the PR. Thanks for your review

**Pull request merged**